seed/php-sdk/unions/src/Union/Types/Shape.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Seed\Union\Types;
+
+use Seed\Core\Json\JsonProperty;
+use Seed\Core\Json\JsonSkip;
+use Seed\Core\Types\DiscriminatedUnion;
+use Seed\Core\Types\Discriminant;
+use Seed\Union\Types\Circle;
+use Seed\Union\Types\Square;
+
+class Shape extends DiscriminatedUnion
+{
+    /**
+     * @var 'circle'|'square'|'_unknown' $type 
+     */
+    #[JsonProperty('type')]
+    #[Discriminant(['circle' => Circle::class, 'square' => Square::class,])]
```

Comment:
nit: Trailing comma here:
```suggestion
    #[Discriminant(['circle' => Circle::class, 'square' => Square::class])]
```

Reply:
done